test cases/java/9 jdk/meson.build

```diff
@@ -0,0 +1,14 @@
+project('jdkjava', ['c', 'java'])
+
+if build_machine.system() == 'windows' and build_machine.cpu_family() == 'x86'
+  error('MESON_SKIP_TEST: failing builds on 32bit Windows because a 32bit JDK isn't available in the Azure Pipelines Windows images')
```

Comment:
I think you need to escape the `'` in `isn't`

Reply:
Fixed.